### ISSUE TITLE
Docs Docs Docs - We need docs

### ISSUE THREAD

**Issue by lwasser** [earthlab/earthpy#8]
We have none. and we desperately need to begin documenting this package. This is the start of this effort... 

1. setup readthe docs account
2. setup docs folder
3. hopefully @betatim can help us with an autobuild for this to make life easier!
4. maybe @jlpalomino  can help me and we can get maybe joe even to pitch in to help with just a  bit of documentation? or an intern could do this too?

**Comment by betatim** [comment]
Do you want to pick a name for the read the docs subdomain we want to use? I'd suggest `earthpy`.

Once we have that I can get the basics setup on RTD and in the repo.

**Comment by lwasser** [comment]
hey @betatim  are you back? i haven't wanted to bother you. i have so many questions. 
earthpy.readthedocs.io? that sounds good to me.
I'd really like to understand how to setup an auto build. 
I also am having some docker image issues with my ea-hub and wanted to figure out how to test things. hoping you can help me not make bad decisions.

**Comment by lwasser** [comment]
heya @betatim -- 
I made this commit ... is travis setup to build the docs? Or can you kindly help me understand how the docs are built and how i would setup travis if it's not in place?

https://github.com/earthlab/earthpy/commit/dbb5c8fc3e6483c0a6de22376a26bc02f8b5b3e6 

I pretty much want what we have for hub-ops! 

I just noticed that this page did actually update. So is it the case that travis isn't involved in building here? readthedocs does the build?

https://earthpy.readthedocs.io/en/latest/index.html 
Thank you for any clarification!!

**Comment by betatim** [comment]
Correct. Travis doesn't get involved as RTD notices when something changes and rebuilds the documentation for us.

**Comment by lwasser** [comment]
awesome. thank you so so much @betatim  . it's working i just didn't understand what was going on.

**Comment by betatim** [comment]
A case of being so deep into the tech side of things that I didn't even realise that not everyone knows RTD and its magic inside out :)

Just now I did add a PR to get travis involved (#11). This is purely to get us a build of the docs during a PR so we can see if the docs build or not before merging a PR. There are however (😢) cases when travis will succeed and RTD fails (super rare but does seem to happen, mostly it seems because subtle differences between the environments). I also made a PR to add a badge to the README where we can see the status of the RTD build (#10).
